examples/advanced/k8s_addons/main.tf

```diff
@@ -308,8 +308,11 @@ module "kubernetes-addons" {
   #---------------------------------------
   # COMMUNITY PROMETHEUS ENABLE
   #---------------------------------------
-  enable_prometheus = true
+  # Amazon Prometheus Configuration to integrate with Prometheus Server Add-on
+  enable_amp_for_prometheus      = true
```

Comment:
don't think you need this property right? It is implied by the presence of the workspace_id?